src/2_RTB/VerbaliEsterni/VerbaleEsterno_061223/content.typ

```diff
@@ -0,0 +1,83 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro, company
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+#let participants_company = csv("participants_company.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+== Partecipanti di #team
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+== Partecipanti di #emph[#company]
+#for member in participants_company.flatten() [
+  - #member
+]
+
+= Sintesi Elaborazione Incontro
+
+/*************************************/
+/*    INSERIRE SOTTO IL CONTENUTO    */ 
+/*************************************/
+
+== Esposizione dei progressi raggiunti 
+
+Nell'incontro con la Proponente, il team ha illustrato in dettaglio l'evoluzione del Proof of Concept, evidenziandone gli importanti progressi compiuti nel corso dello sprint. La Proponente ha manifestato un apprezzamento molto positivo per i risultati raggiunti finora, confermando la validità della direzione intrapresa per il successo del progetto.Inoltre, sono stati  dati feedback per migliorare quanto sviluppato, a livello grafico, per rendere più veloce e facile la comprensione dei vari pannelli.
+
+In seguito, la riunione è proseguito con una dettagliata discussione riguardo la parte inerente la documentazione, ponendo particolare enfasi sull'Analisi dei Requisiti.
+
+== Avanzamento del codice        
+ 
+Gli obiettivi prefissati comprendevano l'archiviazione dei dati provenienti da Kafka nel database OLAP ClickHouse, nonché lo sviluppo di una dashboard in Grafana. La dashboard avrebbe dovuto visualizzare una mappa della città indicando la posizione dei sensori di temperatura e presentare in modo chiaro le temperature medie rilevate e un grafico che mostrasse l'andamento sinusoidale della temperatura.
+
+
+== FeedBack della proponente
+
+La Proponente ha sollevato una critica sostanziale in merito alla comprensibilità della dashboard, evidenziando la difficoltà di interpretare il grafico sinusoidale a causa della sovrapposizione di dati. La soluzione proposta è quella di rendere le temperature più singolari e differenziare il modo in cui i sensori generino l'onda sinusoidale, accelerando il ciclo per rappresentare una "giornata" in pochi minuti e aumentare il tasso di generazione dei dati per una rappresentazione più dinamica, inoltre è stato consigliato di aggiungere un pannello descrittivo.
```

Comment:
"accelerando il ciclo per rappresentare una "giornata" in pochi minuti e aumentare il tasso di generazione dei dati per una rappresentazione più dinamica" 

non credo che sia una soluzione alla sovrapposizione dei dati.